### PR TITLE
#205: JoinedText

### PR DESCRIPTION
**Pull request by fabriciofx**
As per #205 :

`JoinedText` introduced.

### CODE REVIEW

**Comment by 0crat** [comment]
@fabriciofx I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/test/java/org/cactoos/text/JoinedTextTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * Test case for {@link JoinedText}.
+ * @author Fabricio Cabral (fabriciofx@gmail.com)
+ * @version $Id$
+ * @since 0.9
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class JoinedTextTest {
+
+    @Test
+    @SuppressWarnings("PMD.AvoidDuplicateLiterals")
```
Comment (yegor256) [inline src/test/java/org/cactoos/text/JoinedTextTest.java]: @fabriciofx what is the point of using duplicate literals? Why can't we use different names in these test methods?

Comment (fabriciofx) [inline src/test/java/org/cactoos/text/JoinedTextTest.java]: @yegor256 I removed the duplicated literals. Thanks for your attention!

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**